### PR TITLE
add recent patients

### LINKED ISSUE #740: Recent patients
### Describe the feature

The client can retrieve the recent patients by [this new endpoint](https://proto.helpwave.de/#proto.services.task_svc.v1.GetRecentPatientsRequest). The new `RecentPatientCard` should contain similar information as a row in the patient list.

- Name of the patient
- Room and bed

The on-click behavior should navigate the user diretly to the patient on the ward overview. Right now, the ward_id is not part of the response. Until this issue is resolved, please fetch the ward_id after the on-click via https://proto.helpwave.de/#proto.services.task_svc.v1.GetRoomRequest.

### Why is this feature needed?

For faster navigation

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by DasProffi**
## Which issues does this pull request close?
closes #740

### CODE REVIEW

File: tasks/mutations/room_mutations.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { useMutation, useQuery, useQueryClient } from '@tanstack/react-query'
 import {
   CreateRoomRequest,
   DeleteRoomRequest,
-  GetRoomOverviewsByWardRequest,
+  GetRoomOverviewsByWardRequest, GetRoomRequest,
```
Comment (use-to) [inline tasks/mutations/room_mutations.ts:5]:
```suggestion
  GetRoomOverviewsByWardRequest,
  GetRoomRequest,
```

File: tasks/mutations/patient_mutations.ts [hunk 2]
```diff
@@ -13,12 +13,12 @@ import {
   GetPatientAssignmentByWardRequest,
   GetPatientListRequest,
   DeletePatientRequest,
-  ReadmitPatientRequest
+  ReadmitPatientRequest, GetRecentPatientsRequest
```
Comment (use-to) [inline tasks/mutations/patient_mutations.ts:16]: Please fix your formatter.

```suggestion
  ReadmitPatientRequest,
  GetRecentPatientsRequest
```

**Review by use-to — CHANGES_REQUESTED** [review]
Currently unable to test it against the backend. It just returns an empty set of patients.
Also, please consider removing the organization sections (so only recent patient and wards will be shown). It not so important to navigate to the organization overview page all the time.

**Review by MaxSchaefer — CHANGES_REQUESTED** [review]
I've debugged this. Seams like services on Staging (and probably on production as well) has problems with the redis interaction.

For now, I added a recent patient by hand that you can move. `GetRecentPatients` is now returning one recent patient. The frontend does load and then fails silenty, can you please check this @DasProffi?

**Comment by MaxSchaefer** [comment]
@DasProffi linter 👀

**Comment by MaxSchaefer** [comment]
@DasProffi please re-request my review when the requested changes got addressed. Before we merge this, we need to hide the recent patients until https://github.com/helpwave/services/issues/458 is resolved.

**Comment by MaxSchaefer** [comment]
> @DasProffi please re-request my review when the requested changes got addressed. Before we merge this, we need to hide the recent patients until [helpwave/services#458](https://github.com/helpwave/services/issues/458) is resolved.

@DasProffi EDIT: We can hide the title on an empty response instead.

**Comment by DasProffi** [comment]
> @DasProffi EDIT: We can hide the title on an empty response instead.

Problem is addressed, the query update once the backend is fixed will be done in #754

**Review by MaxSchaefer — APPROVED** [review]

**Comment by MaxSchaefer** [comment]
@use-to requested changes :)

**Review by use-to — APPROVED** [review]

**Pull request merged**